source/common/tls/utility.cc

```diff
@@ -192,22 +192,20 @@ std::vector<std::string> Utility::getSubjectAltNames(X509& cert, int type, bool
 
 std::string Utility::generalNameAsString(const GENERAL_NAME* general_name) {
   std::string san;
+  ASN1_STRING* str;
```

Comment:
Changes in this file seem unrelated. Revert? Or if you like them, `ASN1_STRING* str = nullptr;` to avoid possibility of uninitialized pointer in future changes.

Reply:
yeah this was a coverage improvement to make up for not e2e testing the factory creation failing.  I could alternately add a unit test and not link that in but seems overkill.